infrastructure/src/main/java/org/corfudb/infrastructure/HandlerMethods.java

```diff
@@ -207,10 +207,14 @@ private void registerMethod(@Nonnull final MethodHandles.Lookup caller,
     // Create a timer using cached timer name for the corresponding type
     private Optional<Timer> getTimer(@Nonnull CorfuMsgType type) {
         timerNameCache.computeIfAbsent(type,
-                                       aType -> (CorfuComponent.INFRA_MSG_HANDLER +
-                                                 aType.name().toLowerCase()));
+                aType -> (CorfuComponent.INFRA_MSG_HANDLER +
+                        aType.name().toLowerCase()));
+        double[] percentiles = new double[]{0.50, 0.95, 0.99};
 
         return MeterRegistryProvider.getInstance()
-                .map(registry -> Timer.builder(timerNameCache.get(type)).register(registry));
+                .map(registry -> Timer.builder(timerNameCache.get(type))
+                        .publishPercentiles(percentiles)
+                        .publishPercentileHistogram(true)
+                        .register(registry));
```

Comment:
Should these updates also be made to the `RequestHandlerMethods` class? This class is only responsible for `CorfuMsg` types and does not take into account the server handler methods that handle Protobuf messages.

Reply:
Did not know about RequestHandlerMethods. I'll address in a separate PR, because this one is mostly to remove dependencies on logback.